### ISSUE TITLE
custom keybindings for suggestions with autocomplete disabled doesn't work

### ISSUE THREAD

**Issue by evilgeniuscreative** [sourcegraph/cody#744]
- **Sourcegraph version:06.6** 
- **Platform information: Mac Ventura 13.5** 

#### Steps to reproduce:

1. Asked Cody how to set up suggestions on command by Key combination
2. Cody suggested "cody.inlineSuggestions": false  and adding a shortcut to keybindings.json:     
{
        "key": "ctrl+alt+right",
        "command": "editor.action.triggerSuggest"
    }

#### Expected behavior:
1. Cody would not make suggestions automatically
2. Cody would make suggestions when I used the ctrl+alt+right key combination

#### Actual behavior:
1. Cody continued to make automatic suggestions
2. Key combination never worked

If you would like immediate help on this, please email support@sourcegraph.com (you can still create the issue, but there are no [SLAs](https://about.sourcegraph.com/support/) on issues like there are for support requests).

**Comment by jdorfman** [comment]
Thanks @evilgeniuscreative for the report and detailed explanation. The team will take a look asap.

**Comment by evilgeniuscreative** [comment]
@jdorfman thank you. Having it be an option for users to use a key combination to get an in line suggestion would be really useful for both the users and for source graph because you wouldn't get 1, million API calls from non-paying customers who don't really want all those API calls. Also not all of the in-line suggestions are good to sometimes a bad one can accidentally get inserted into the code if you press the wrong button by accident. The way I look at it it's a win-win if this were a functionality. I thought I suggested this as an idea but I haven't been able to find it by searching for anything that I wrote here
